androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/WootricRemoteRequestTask.java

```diff
@@ -83,8 +105,8 @@ protected String doInBackground(Void... params) {
 
     @Override
     protected void onPostExecute(String response) {
-        Log.d("WOOTRIC_SDK - response", response);
         String decodedString = "";
+        Log.d("WOOTRIC_SDK", "response: " + response);
```

Comment:
@hachibu this line. Sometimes `response` was `null` so it was crashing on `Log`. I'm surprised/annoyed this happens but c'est la vie. By doing `"response: " + response` the second parameter passed to `Log.d()` will never be null.